administrator/components/com_localise/Model/PackageModel.php

```diff
@@ -1331,4 +1333,157 @@ public function uploadOtherFile($file, $location)
 
 		return true;
 	}
+
+	/**
+	 * Method to get the HTML output required to update the translations field list by the selected language tag.
+	 *
+	 * @return   string  The data for the jform_translations field.
+	 */
+	public function updateTranslationsList($data)
+	{
+		$params = ComponentHelper::getParams('com_localise');
+		$reftag	= $params->get('reference', '');
+
+		if (empty($reftag))
+		{
+			$reftag = 'en-GB';
+		}
+
+		// Getting the data from ajax call
+		$packagename = htmlspecialchars($data[0]->packagename);
+		$langtag     = htmlspecialchars($data[0]->languagetag);
+
+		// Initiating form instance
+		$filepath = JPATH_ADMINISTRATOR . "/components/com_localise/forms/package.xml";
+		$form_package = Form::getInstance("package", $filepath, array("control" => "jform"));
+
+		//Form::addFieldPath(JPATH_ADMINISTRATOR . '/components/com_localise/Field');
+		//Form::addFormPath(JPATH_ADMINISTRATOR . '/components/com_localise/forms');
+
+		// Adding the params bellow at translations field only when Ajax call.
```

Comment:
bellow => below

`// Adding the params below for the translations field only when Ajax call.`

Reply:
Done